### ISSUE TITLE
When you try to restart the monitor, nothing happens. (VSC-284)

### ISSUE THREAD

**Issue by ArTyGill** [espressif/vscode-esp-idf-extension#81]
When you try to restart the monitor, nothing happens.

Workaround to run the IDF build, flash, monitor command.

**Comment by ArTyGill** [comment]
OS: Windows 10 1909
IDF-PATH: release/4.0 (master)
VScode: version 1.44.0

**Comment by meltdown03** [comment]
I have this issue also. Nothing is logged, no error is displayed. It only happens if I close the initial terminal that is opened on the first Monitor launch.

OS: Archlinux
ESP-IDF: master branch 4.2
All packages up-to-date

**Comment by SoumeshBanerjee** [comment]
> When you try to restart the monitor, nothing happens.

How do you restart the monitor, is it just launching the monitor clicking the monitor icon at the bottom and then closing the monitor terminal and then launching monitor again by clicking the button ?

**Comment by brianignacio5** [comment]
In the log it seems you have  `idf.pythonBinPath is not defined` Do you configure the extension using `ESP-IDF: Configure extension` command ?

**Comment by meltdown03** [comment]
That's what I do. I also press ctrl-] to stop the monitor before closing the terminal. The tasks all work everytime, so do the other icons, just not the monitor icon.

**Comment by SoumeshBanerjee** [comment]
@meltdown03 Thanks for the feedback, we found the issue and working to fix the same

**Comment by SoumeshBanerjee** [comment]
Possible duplicate of #77

**Comment by ArTyGill** [comment]
@SoumeshBanerjee Yes, the same problem, the only option that works well is the "build, flash, monitor" button, which displays the message "idf monitor was closed" before starting a new task.
